lenskit-core/src/main/java/org/lenskit/results/BasicResult.java

```diff
@@ -0,0 +1,94 @@
+/*
+ * LensKit, an open source recommender systems toolkit.
+ * Copyright 2010-2014 LensKit Contributors.  See CONTRIBUTORS.md.
+ * Work on LensKit has been funded by the National Science Foundation under
+ * grants IIS 05-34939, 08-08692, 08-12148, and 10-17697.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU General Public License for more
+ * details.
+ *
+ * You should have received a copy of the GNU General Public License along with
+ * this program; if not, write to the Free Software Foundation, Inc., 51
+ * Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
+ */
+package org.lenskit.results;
+
+import com.google.common.base.MoreObjects;
+import org.lenskit.api.Result;
+
+import javax.annotation.concurrent.Immutable;
+import java.io.*;
+
+/**
+ * A basic {@link Result} implementation with no details.
+ *
+ * @see Results
+ */
+@Immutable
+public final class BasicResult extends AbstractResult implements Serializable {
+    private static final long serialVersionUID = 1L;
+
+    /**
+     * Create a new basic result.
+     * @param id The result ID.
+     * @param score The result score.
+     * @see Results#create(long, double)
+     */
+    public BasicResult(long id, double score) {
+        super(id, score);
+    }
+
+    private void writeObject(ObjectOutputStream out) throws IOException {
```

Comment:
Yes. AbstractResult is not Serializable, but BasicResult is, so it needs some help.